egov-indexer/pom.xml

```diff
@@ -5,12 +5,12 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>1.5.22.RELEASE</version>
+    <version>2.2.6.RELEASE</version>
     <relativePath></relativePath>
   </parent>
   <groupId>org.egov</groupId>
   <artifactId>egov-indexer</artifactId>
-  <version>0.0.1-SNAPSHOT</version>
+  <version>0.1.0-SNAPSHOT</version>
```

Comment:
why is this `0.1.0`? Minimum version of any service in production has to be `1.0.0`